style.css

```
@@ -0,0 +1,488 @@
+
+html, body {
+    width: 100%;
+}
+
+*, *::before, *::after {
```

This block of code is used by most developers to stop manually margin/paddings, and also allow border box sizing. The wild card selector selects everything - can't see need for before or after at first glance so just for your consideration.

```suggestion
* {
    box-sizing: border-box;
    margin: 0;
    padding:0;
}
```